ext/oj/parse.c

```diff
@@ -263,6 +263,10 @@ read_escaped_str(ParseInfo pi, const char *start) {
 		}
 		break;
 	    default:
+		if (CompatMode == pi->options.mode) {
+		    buf_append(&buf, *s);
+		    break;
+		}
```

Comment:
Can remove the special case for `\'` now. Should also add a similar comment here.


	// The json gem claims this is not an error despite the
	// ECMA-404 indicating it is not valid.